File: Python3/1165 - Número Primo.py
N = int(input())
for i in range(N):
    CONTADOR = 0
    X = int(input())
    for j in range(X):
        if X % (j + 1) == 0:
            CONTADOR += 1
    if CONTADOR == 2:
        print('{} eh primo'.format(X))
    else:
        print('{} nao eh primo'.format(X))

File: CSharp/1165 - Número Primo.cs
using System; 

class URI {

    static void Main(string[] args) { 

        bool primo = true;
    int N = int.Parse(Console.ReadLine()), X = 0;

		for(int i = 0; i < N; i++) {

			X = int.Parse(Console.ReadLine());
			primo = true;

      for (int j = 2; j < X; j++) {

        if (X % j == 0)
          primo = false;

      }

      if (primo == true) 
        Console.WriteLine("{0} eh primo", X);
      else 
        Console.WriteLine("{0} nao eh primo", X);
	
    }

    }

}